VERSION

```diff
@@ -1 +1 @@
-1.3.7
+1.3.8
```

Comment:
I don't think we need to bump the version

Reply:
changing back to 1.3.7